lib/discovery.js

```diff
@@ -186,6 +186,7 @@ function mixinDiscovery(MySQL, mysql) {
         (table ? ' WHERE table_name=' + mysql.escape(table) : ''),
       'table_name, ordinal_position', {});
     }
+    sql += ' ORDER BY ordinal_position';
```

Comment:
Why do we need this ? What's the issue happening without it ?

Reply:
I have a use case where the frontend generates a table based on the model generated - which is expected to be in the order based on DB structure (in ordinal order).

We can possibly have an option that we can pass on to the connector to have the model properties in ordinal or not.

Reply:
yes, lets make it configurable as an option